azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/manager/CaptionsRttDataManager.kt

```diff
@@ -216,6 +216,15 @@ internal class CaptionsRttDataManager(
     }
 
     private suspend fun handleCaptionData(newCaptionsRecord: CaptionsRttRecord) {
+        val lastFinalRtt = captionsAndRttMutableList
+            .lastOrNull { it.type == CaptionsRttType.RTT && it.isFinal }
```

Comment:
I think we also need to include a user id, as last final RTT can be from other user.